### ISSUE TITLE
Explicit units for measurements and calculations

### ISSUE THREAD

**Issue by jpas** [UofSSpaceTeam/roveberrypy#82]

**Comment by ottopasuuna** [comment]
Right, at the very least these should be documented better. I'll go through the code base this evening to update more of the documentation, there are other things that need updating as well.
Do you know of a good way to associate units with values in python? I'd rather not have to use a third party library just for that.

**Comment by wjtan360-git** [comment]
You could create a class that contains two instance variables: one for the numerical value, and one for the unit of measurement.

**Comment by jpas** [comment]
That is part of what I had in mind, we could also implement the standard mathematical operations for these unit classes.
